mudirac/mudirac.xml

```diff
@@ -185,9 +205,24 @@ spec_expdec: $spec_expdec
             <option value="SPHERE"> finite size, uniformly charged spherical nucleus </option>
             <option value="FERMI2"> Fermi 2-term charge distribution </option>
         </param>
-        <param name="uehling_correction" type="boolean" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not" optional="true"/>
+        <conditional name="uehling_correction">
+            <param name="uehling_correction" type="select" display="radio" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not">
+                <option value="false" selected="true">False</option>
+                <option value="true">True</option>
+            </param>
+            <when value="false"/>
+            <when value="true">
+                <param name="uehling_steps" type="integer" value="" label="integration steps for the Uehling potential." help=" Higher numbers will make the Uehling energy more precise but increase computation times. Default is 100." optional="true"/>
```

Comment:
```suggestion
                <param name="uehling_steps" type="integer" value="100" min="0" label="integration steps for the Uehling potential." help=" Higher numbers will make the Uehling energy more precise but increase computation times. Default is 100."/>
```

We need a minimum value here, as setting to a negative number results in a vague error:
```
terminate called after throwing an instance of 'std::bad_alloc' what(): std::bad_alloc /home/kez92691/muon-galaxy/database/jobs_directory/000/640/tool_script.sh: line 10: 4011330 Aborted (core dumped) mudirac input.in
```
This should also be properly handled by the underlying mudirac code, but the best place to prevent it in Galaxy is here (before we even submit the job).